openhands/agenthub/codeact_agent/codeact_agent.py

```diff
@@ -177,6 +175,33 @@ def get_action_message(
                 tool_calls=assistant_msg.tool_calls,
             )
             return []
+        elif isinstance(action, AgentFinishAction):
+            role = 'user' if action.source == 'user' else 'assistant'
```

Comment:
else 'agent'?

Reply:
It's the role?

Reply:
ohhhhhh you are absolutely right! my brain somehow stopped working when it passes 10 pm now ;(